tools/docker/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM node:10.14 as base
```

Comment:
should this be `node:10.14-alpine` instead?

Reply:
done